package.json

```diff
@@ -28,6 +29,8 @@
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/github": "^5.1.1",
+    "@popperjs/core": "^2.11.6",
+    "@rushstack/eslint-patch": "^1.2.0",
```

Comment:
What are these being used for?

Reply:
This is an eslint patch that resolves some eslint bugs in typescript.